frontend/pulse3d/src/components/basicWidgets/InputDropdownWidget.js

```diff
@@ -9,7 +9,7 @@ export default function InputDropdownWidget({
   label,
   reset,
   handleSelection,
-  defaultFile,
+  defaultFileIndex,
```

Comment:
should remove use of the word file in general to make it reusable to components without file names as the options